internal/testhelper/ci-ar-cleanup-policy.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "name": "delete-old-ci-images",
+      "action": {"type": "Delete"},
+      "condition": {
+        "olderThan": "1d"
```

Comment:
Is 1d a little bit short? Probably 3 or 4 days

Reply:
changed to 3days.